lang/ilias_de.lang

```diff
@@ -13542,6 +13542,10 @@ iass#:#iass_notify#:#Teilnehmer nach abschließender Bewertung benachrichtigen
 iass#:#iass_finalize#:#Abschließen
 iass#:#il_iass_member#:#Teilnehmer
 iass#:#il_iass_members#:#Teilnehmer
+iass#:#iass_event_time_place_required#:#Zeit und Ort sind Pflichtangaben
+iass#:#iass_event_time_place_required_info#:#Zeit und Ort müssen mit angegeben werden.
+iass#:#iass_file_required#:#Datei ist eine Pflichtangabe
+iass#:#iass_file_required_info#:#Eine Datei muss beiligen.
```

Comment:
This does not relate to the original ticket, right?

Reply:
Thats right. Should i remove it?

Reply:
Nope, boy- or girlscouting imo is always fine. Hints on that kind of stuff could go to the description of the PR to make it clear that this is in fact not about the bug.

Reply:
Just my 5 cent... It is "beiliegen" and not "beiligen". And all four new lang vars are real sentences (subject, verb, object) and should end with a dot.
(Sorry for just adding a comment and not offering a PR.)

Reply:
Thanks. I will correct this immediately.